docs/self-hosting/migration.md

```diff
@@ -1,5 +1,9 @@
 # Migrating from Older Versions
 
+### v0.5.0
+The fronted is not required any more `tombursch/kitchenowl:latest` is now an all-in-one container that hosts the web application and backend. Take a look at the [install instructions](index.md) if you want to switch your setup.
+Existing setups don't need to be adapted, they will still work like they used to. Although, there now exists a docker images `tombursch/kitchenowl-backend:latest` specifically designed for the split setup.
```

Comment:
I am not sure how existing setups can survive without manual migration.

Afaik, `JWT_SECRET_KEY` was previously configured on the "backend" container. And with kitchenowl now becoming the all-in-one container, the `kitchenowl` container can not really read the env variable defined for the `backend` container.

I am not 100% sure if I am right on this one, so feel free to correct me :)

Reply:
Thanks for the feedback! :)
> `JWT_SECRET_KEY` was previously configured on the "backend" container.
Yes, but there didn't exist an explicit backend container image before. `tombursch/kitchenowl:latest` was the backend container and now is the aio container. The variables are applicable to both. If you have an old setup, you just redundantly serve the web app (with front and backend), but it shouldn't really matter. The aio container will still serve the uwsgi protocol on port 5000, required for `tombursch/kitchenowl-web`.